libmambapy/CMakeLists.txt

```diff
@@ -7,20 +7,16 @@ cmake_minimum_required(VERSION 3.18.2)
 
 include("../cmake/CompilerWarnings.cmake")
 
-cmake_policy(SET CMP0025 NEW)
-cmake_policy(SET CMP0077 NEW)
-cmake_policy(SET CMP0057 NEW)
-
 project(libmambapy)
 
 if(NOT TARGET mamba::libmamba)
-    find_package(libmamba REQUIRED)
+    find_package(libmamba CONFIG REQUIRED)
     set(libmamba_target mamba::libmamba-dyn)
 else()
     set(libmamba_target mamba::libmamba)
 endif()
 
-find_package(Python COMPONENTS Interpreter Development)
+find_package(Python COMPONENTS Interpreter Development.Module)
```

Comment:
The default `Development` looks for the `Embed` target too, which isn't required and sometimes (like on manylinux) not available.